src/quo/components/inputs/input/view.cljs

```diff
@@ -55,16 +55,17 @@
 
 (def ^:private custom-props
   "Custom properties that must be removed from properties map passed to InputText."
-  [:type :blur? :theme :error? :right-icon :left-icon :disabled? :small? :button
+  [:type :blur? :error? :right-icon :left-icon :disabled? :small? :button
    :label :char-limit :on-char-limit-reach :icon-name :multiline? :on-focus :on-blur
    :container-style])
 
 (defn- base-input
-  [{:keys [blur? theme error? right-icon left-icon disabled? small? button
```

Comment:
if we're removing `theme` prop from here and relying on theme value from `(quo.theme/use-theme-value)` should we also modify usage of `input` component where theme value is directly passed?
example : https://github.com/status-im/status-mobile/blob/b4d27d287a5383f515a1835e647f2c5670adc319/src/status_im/contexts/profile/edit/name/view.cljs#L48

Reply:
that's actually interesting, it should be override-theme i guess

Reply:
oh that's probably was implemented wrong because of this bug :) so someone just passed :dark :)

Reply:
fixed thanks